src/models/ysz_model_new_prms.jl

```diff
@@ -1,4 +1,4 @@
-module ysz_model_fitted_parms
+module ysz_model_new_prms
```

Comment:
can either src/models/ysz_model_fitted_parms.jl or src/models/ysz_model_new_prms.jl deleted? They are fairly identical.

Reply:
Actually, there is one difference ... x_frac = 0.08 vs x_frac = 0.13 
These two separate files were introduced, when I started to deal with new material > monocrystaline YSZ. And I want to store the new set of material parameters in the new file. But both of this files can be deleted, because **ysz_model_GAS_LoMA.jl** is now a generalisation of all the previous models. And storing of different sets of fitted parameters may be done somewhere else ... maybe in a separate file.